test/parallel/test-internal-modules-expose.js

```diff
@@ -5,3 +5,4 @@ require('../common');
 const assert = require('assert');
 
 assert.strictEqual(typeof require('internal/freelist').FreeList, 'function');
+assert(!('_exposeInternals' in process), 'no process property is leaked');
```

Comment:
@jasnell PTAL, I deliberately am making  no change to  the process properties (for cleanliness, and because it may potentically be backwards incompat, and also because its unnecessary, we can find if internal modules were exposed pretty easily by just trying to require one and seeing what happens, which is unusual to want to know, but possible).